src/UI/Components/Diagram/Canvas.test.tsx

```diff
@@ -413,7 +413,10 @@ describe("Canvas.tsx", () => {
       await user.click(copyButton);
     });
 
-    const clipboardText = await navigator.clipboard.readText();
+    const clipboardItems = await navigator.clipboard.read();
```

Comment:
what does this change relates to?

Reply:
polyfills, it broke that part of the text